game-store-landing-page/test-footer/index.html

```diff
@@ -0,0 +1,62 @@
+    <!DOCTYPE html>
```

Comment:
What is the directory test footer for?

Reply:
was testing the footer for different screen sizes hence a separate file, will remove it now